notebooks/GoPro2CropKMZ.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>I run into an error here.</p><p> </p><p>Error detailed below:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">HttpError                                 Traceback (most recent call last)
/usr/local/lib/python3.10/dist-packages/ee/data.py in _execute_cloud_call(call, num_retries)
    401   try:
--&gt; 402     return call.execute(num_retries=num_retries)
    403   except googleapiclient.errors.HttpError as e:

</pre><p> </p><p> </p><p>8 frames</p>

<pre class="ql-syntax" spellcheck="false">HttpError
: &lt;HttpError 403 when requesting https://earthengine.googleapis.com/v1/projects/bsos-geog-harvest1/algorithms?prettyPrint=false&amp;alt=json returned "Caller does not have required permission to use project bsos-geog-harvest1. Grant the caller the roles/serviceusage.serviceUsageConsumer role, or a custom role with the serviceusage.services.use permission, by visiting https://console.developers.google.com/iam-admin/iam/project?project=bsos-geog-harvest1 and then retry. Propagation of the new permission may take a few minutes.". Details: "[{'@type': 'type.googleapis.com/google.rpc.Help', 'links': [{'description': 'Google developer console IAM admin', 'url': 'https://console.developers.google.com/iam-admin/iam/project?project=bsos-geog-harvest1'}]}, {'@type': 'type.googleapis.com/google.rpc.ErrorInfo', 'reason': 'USER_PROJECT_DENIED', 'domain': 'googleapis.com', 'metadata': {'consumer': 'projects/bsos-geog-harvest1', 'service': 'earthengine.googleapis.com'}}]"&gt;
During handling of the above exception, another exception occurred:

EEException                               Traceback (most recent call last)
/usr/local/lib/python3.10/dist-packages/ee/data.py in _execute_cloud_call(call, num_retries)
    402     return call.execute(num_retries=num_retries)
    403   except googleapiclient.errors.HttpError as e:
--&gt; 404     raise _translate_cloud_exception(e)  # pylint: disable=raise-missing-from
    405 
    406 

</pre><p>EEException: Caller does not have required permission to use project bsos-geog-harvest1. Grant the caller the roles/serviceusage.serviceUsageConsumer role, or a custom role with the serviceusage.services.use permission, by visiting <a href="https://console.developers.google.com/iam-admin/iam/project?project=bsos-geog-harvest1" target="_blank">https://console.developers.google.com/iam-admin/iam/project?project=bsos-geog-harvest1</a> and then retry. Propagation of the new permission may take a few minutes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>